src/core/api/config/axios/responseInterceptor.ts

```diff
@@ -0,0 +1,33 @@
+import { AxiosError, AxiosResponse } from 'axios';
+// import { getCookie } from '../../../../utils/cookie';
+// import { refreshAccessToken } from '../../services/auth';
+
+enum HTTP_STATUS {
+    SUCCESS = 200,
+    INFORMATION = 300,
+    BAD_REQUEST = 400,
+    NOT_FOUND = 404,
+    FORBIDDEN = 403,
+    SERVER_ERROR = 500,
+    UNAUTHORIZED = 401,
+}
+
+export async function onResponseError(error: AxiosError): Promise<AxiosError> {
+    // if (error.response?.status === HTTP_STATUS.UNAUTHORIZED) {
+    //     const { config } = error;
+    //     const currentRefreshToken = getCookie('refershToken');
+    //     if(currentRefreshToken) {
```

Comment:
Getting new access token with current refresh token is not handled.

Reply:
Updated.